pyproject.toml

```diff
@@ -49,6 +51,13 @@ echo run unit tests
 pytest -s ./tests/test_mock_ts.py
 """
 
+[tool.poe.tasks.test]
```

Comment:
not clear how this is different from the iid and ts tests

Reply:
I think I can delete the original iid and ts tasks, I combined them in conftest.py test so that the coveralls test can run them at once.

Reply:
sounds good :+1: